docs/csharp/whats-new.md

```diff
@@ -0,0 +1,32 @@
+---
+title: What's New in C# | C# Guide
+description: How is the C# language evolving
+keywords: C#, Latest Features, What's New, Roslyn
+author: BillWagner
+manager: wpickett
+ms.date: 08/25/2016
+ms.topic: article
+ms.prod: visual-studio-dev-14
+ms.technology: devlang-csharp
+ms.devlang: csharp
+ms.assetid: 77deec51-a14d-46d4-9bb3-faf449477149
+---
+
+# What's new in C#
+
+
+* [C# 7](csharp-7.md):
+    - This page describes the latest features in the C# language. This covers C# 7, currently available in [Visual Studio 15 Preview 4](https://www.visualstudio.com/en-us/downloads/visual-studio-next-downloads-vs.aspx).
+
+* [C# 6](csharp-6.md):
```

Comment:
do we plan to add more content to this charp-6 page? Otherwise, I'd just kill it and link directly to the Roslyn repo from here.